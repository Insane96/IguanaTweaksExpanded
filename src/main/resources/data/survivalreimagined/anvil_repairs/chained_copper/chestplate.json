{
	"item_to_repair": {
		"id": "survivalreimagined:chained_copper_chestplate"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:copper_ingot"
			},
			"amount": 6
		}
	]
}