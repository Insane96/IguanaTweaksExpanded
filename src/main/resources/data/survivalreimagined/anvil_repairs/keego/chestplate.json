{
	"item_to_repair": {
		"id": "survivalreimagined:keego_chestplate"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:diamond"
			},
			"amount": 8,
			"max_repair": 0.5,
			"cost_multiplier": 0.5
		},
		{
			"repair_material": {
				"id": "survivalreimagined:keego"
			},
			"amount": 2
		}
	]
}