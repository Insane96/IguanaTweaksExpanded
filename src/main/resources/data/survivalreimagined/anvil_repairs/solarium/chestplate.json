{
	"item_to_repair": {
		"id": "survivalreimagined:solarium_chestplate"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:iron_ingot"
			},
			"amount": 8,
			"max_repair": 0.5,
			"cost_multiplier": 0.5
		},
		{
			"repair_material": {
				"id": "survivalreimagined:solarium_ball"
			},
			"amount": 2
		}
	]
}