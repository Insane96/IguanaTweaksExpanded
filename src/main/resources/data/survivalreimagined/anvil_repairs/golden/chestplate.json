{
	"item_to_repair": {
		"id": "minecraft:golden_chestplate"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:golden_ingot"
			},
			"amount": 8
		},
		{
			"repair_material": {
				"id": "minecraft:golden_nugget"
			},
			"amount": 72
		}
	]
}